lib/membrane_rtmp_plugin/rtmp_server.ex

```diff
@@ -0,0 +1,125 @@
+defmodule Membrane.RTMP.Server do
+  @moduledoc """
+  A simple RTMP server, which handles each new incoming connection.
+  """
+  use GenServer
+
+  require Logger
+
+  alias Membrane.RTMP.Server.ClientHandlerBehaviour
+
+  @enforce_keys [:behaviour, :port, :use_ssl?, :listen_options]
```

Comment:
```suggestion
  @enforce_keys [:client_handler, :port, :use_ssl?, :listen_options]
```

and perhaps `listen_options` shouldn't be configurable

Reply:
done